linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_overflow_action/ansible/shared.yml

```diff
@@ -4,7 +4,13 @@
 # complexity = low
 # disruption = low
 
-{{{ ansible_set_config_file(file="/etc/audit/auditd.conf",
+{{% if product in ["rhel8", "fedora", "ol8", "rhv4"] %}}
```

Comment:
Instead of duplicating this logic for the config file location, I think we should create a variable for it.

Reply:
Hi @Mab879 , would the work on https://github.com/ComplianceAsCode/content/pull/7594 be a better approach?

Reply:
I think I like that better, I have requested some feedback from some others on that PR as well.